recipe/meta.yaml

```
@@ -230,119 +230,121 @@ outputs:
           fi                                                                                  # [unix]
           source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh                    # [unix]
         # Test installation
-        - omnisql -v              # [unix]
-        - which omnisci_initdb    # [unix]
-        - which omnisci_server    # [unix]
-        - omnisql.exe -v          # [win]
+        - heavysql -v              # [unix]
+        - which initheavy    # [unix]
+        - which heavydb    # [unix]
+        - heavysql.exe -v          # [win]
         - |
           if [ -x "$(command -v nvidia-smi)" ]; then  # [unix and cuda_compiler_version not in (undefined, "None")]
-            mkdir data && omnisci_initdb data         # [unix and cuda_compiler_version not in (undefined, "None")]
-            omnisci_server --version                  # [unix and cuda_compiler_version not in (undefined, "None")]
+            mkdir data && initheavy data              # [unix and cuda_compiler_version not in (undefined, "None")]
+            heavydb --version                         # [unix and cuda_compiler_version not in (undefined, "None")]
             rm -rf data                               # [unix and cuda_compiler_version not in (undefined, "None")]
           fi                                          # [unix and cuda_compiler_version not in (undefined, "None")]
-        # these are provided by omniscidb-common package:
-        - which omnisci_insert_sample_data  # [unix]
-        - which startomnisci  # [unix]
+        # these are provided by heavydb-common package:
+        - which heavydb_insert_sample_data  # [unix]
+        - which startheavy  # [unix]
 
     about:
-      home: https://www.omnisci.com/
+      home: https://heavy.ai
       license: Apache-2.0
       license_family: APACHE
       license_file: LICENSE.md
-      summary: The OmniSci database
+      summary: The HeavyDB database
 
       description: |
-        OmniSciDB is an in-memory, column store, SQL relational database
+        HeavyDB is an in-memory, column store, SQL relational database
         that was designed from the ground up to run on GPUs.
 
-        This recipe provides both CUDA enabled and CUDA disabled omniscidb
+        This recipe provides both CUDA enabled and CUDA disabled heavydb
         packages.
-      doc_url: https://www.omnisci.com/docs/latest/
-      dev_url: https://github.com/omnisci/omniscidb
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
 
-  - name: pyomniscidb
+  - name: pyheavydb
     version: {{ version }}
-    script: build-pyomniscidb.sh   # [linux64]
-    script: build-pyomniscidb.bat  # [win]
     build:
       skip: true  # [cuda_compiler_version in ("11.0",)]
-      string: {{ py_build_string }}
       noarch: python
+      script: python -m pip install -vv --no-deps python/.
```

Will this work under Windows? I'd use
```suggestion
      script: python -m pip install -vv --no-deps python
```
if possible.

---

I have no clue, let's try your suggestion.

---

Ah no we cannot! This is trying to install a package which would be named `python` 😅 I will revert for now and since we don't use Windows here it's fine for now. Otherwise we will add another line for windows (e.g. we can just cd before)